socorro/unittest/external/es/base.py

```diff
@@ -301,11 +311,10 @@
         'permissions_needed': [],
         'query_type': 'string',
         'storage_mapping': {
-            'type': 'string',
+            'type': 'text',
             'fields': {
                 'full': {
-                    'type': 'string',
-                    'index': 'not_analyzed',
+                    'type': 'keyword',
```

Comment:
This multi-fields storage serves a simple purpose: when searching, we want to have access to the content separated as terms, and when aggregating, we want to see it as a single block. In this case, take a signature like ``oom | ns::something<T> | randomFunc``. We want it to be stored as ``[oom, ns, something, randomfunc]`` to be able to retrieve it when doing ``<signature> <has terms> <oom>`` (that's why we store it as ``text``) but when showing a list of top signatures, we want to see it as ``oom | ns::something<T> | randomFunc`` (that's why we have a second storage as a ``keyword``, under the ``.full`` key name). Super Search uses the ``has_full_version`` attribute to see if there's a ``.full`` sub-key, and use it in aggregation if so.